src/utils/trust/details.mapper.ts

```diff
@@ -9,8 +9,9 @@ import { addTrustToBeneficialOwner, getTrustByIdFromApp, removeTrustFromBenefici
 const mapDetailToPage = (
   appData: ApplicationData,
   trustId: string,
+  isReview: boolean
 ): Page.TrustDetailsForm => {
-  const trustData = getTrustByIdFromApp(appData, trustId);
+  const trustData = getTrustByIdFromApp(appData, trustId, isReview);
```

Comment:
Lets put the review trust logic you've added to this function into the review trusts utils and put the condition for which util function to call here